include/mosaic_gnss_driver/parsers/sbf/sbf.h

```diff
@@ -106,7 +106,7 @@ namespace sbf {
          * @param crc : The correct CRC value
          * @return : Pass/Fail
          */
-        static bool check_crc(const uint8_t* bytes, size_t length, uint16_t crc);
+        static bool check_crc(const uint8_t* bytes, const size_t length, const uint16_t crc);
```

Comment:
```suggestion
        static bool check_crc(const uint8_t* bytes, size_t length, uint16_t crc);
```
`clang-tidy` warns that const qualification in function declaration has no effect. (essentially when pass by value, caller does not need to worry about `const`).

Reply:
Yeah, it doesn't really make much of a difference here, but not using the `const` qualifier might not allow us to directly pass `const` variables as function arguments. Although it would be rare to do so, I suppose, still adding the `const` would not be harmful either. Again, I don't think it matters very much, so I'll wait for your reply.

Reply:
> but not using the const qualifier might not allow us to directly pass const variables as function arguments
This is only true when you are taking parameters by reference. So for this case the `const` has absolutely no effect in the _declaration_.